src/js/components/ImageDialog.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Button from '@material-ui/core/Button';
+
+class ImageDialog extends Component {
+
+    render() {
+        const { open, onClose } = this.props
+        return(
+            <Dialog
+                open={open}
+                onClose={onClose}
+                aria-labelledby="alert-dialog-title"
+                aria-describedby="alert-dialog-description"
+            >
+                <DialogTitle id="alert-dialog-title">{"Mars"}</DialogTitle>
```

Comment:
please use a variable or remove redundant brackets and quotes

Reply:
okay, to be honest this component need a big improvement, cause it is only prototype with  lot of stubs, and I will improve it right after some clarification about data format